src/components/organisms/Footer/Footer.js

```diff
@@ -33,36 +32,61 @@ const Footer = (props) => {
                 }
               }
             }
-            id
+            socialIcons {
+              icons {
+                title
+                image {
+                  file {
+                    url
+                  }
+                }
+                url
+              }
+            }
           }
         }
       }
     }
   `)
-  const colorHeader = "#FF5E0A"
-  const header = data.header.edges.find(item => item.node.id === props.contentfulId).node
-  const navigationLinks = header.navBar.navigationLinks.map(item => {
+  const colorFooter = "#FF5E0A"
+  const footer = data.footer.edges.find(
+    item => item.node.id === props.contentfulId
+  ).node
+  const navigationLinks = footer.navBar.navigationLinks.map(item => {
     return { caption: item.caption, slug: item.to.slug }
   })
-  const languageOptions = header.languageSelector.options.map(
-    item => {
-      return { text: item, value: item }
-    }
-  )
-  const actionImage = {imageUrl: header.navBar.mainIcon.image.file.url, slug: header.navBar.mainIcon.to.slug}
+  const SocialIcons = footer.socialIcons.icons.map(item => {
+    return { slug: item.title, imageUrl: item.image.file.url, linkTo: item.url }
+  })
+  const actionImage = {
+    imageUrl: footer.navBar.mainIcon.image.file.url,
+    slug: footer.navBar.mainIcon.to.slug,
+  }
 
   return (
-    <AppBar position="static" style={{ background: colorHeader }}>
+    <AppBar
+      position="relative"
+      justifyContent="center"
+      style={{
```

Comment:
I think we have to options here or we just use in all the app .scss files or we use the useStyle or makeStyle of Material UI. I think is better avoid send styles like here and also avoid the most that we can use `px`.

Reply:
Yesterday I was trying to use **makestyles**. Something like:
```
  footer: {
    padding: theme.spacing(1),
    position: "fixed",
    bottom: 0,
  },
```
But It didn't work. Maybe we could peer programming to analyze the options

Reply:
Sure!, just take into account for your next PR that is better to use the **makeStyles** option.